python_ta/transforms/type_inference_visitor.py

```diff
@@ -442,6 +442,13 @@ def visit_call(self, node: astroid.Call) -> None:
     def visit_binop(self, node: astroid.BinOp) -> None:
         left_inf, right_inf = node.left.inf_type, node.right.inf_type
 
+        if isinstance(left_inf, TypeFail):
```

Comment:
`_arithm_convert` already has the accept failable; why don't you branch on `isinstance(arithm_type, TypeFail)` below